rubix-hadoop2/pom.xml

```diff
@@ -27,6 +27,12 @@
             <version>2.6.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-azure</artifactId>
+            <version>2.7.0</version>
```

Comment:
Is it compatible with hadoop-core 2.6.0? Were you able to access Blobstore with these changes?

Reply:
I had some problem staring a azure cluster. I am planning to test this out and update here

Reply:
I have tested this patch in our hadoop2 cluster with hive query.

Reply:
Is it possible to provide a node bootstrap that Qubole Presto QA can test with ? Presto support is paramount.

Reply:
Yes. We can provide a node bootstrap to test this.

Reply:
I have tested this in a presto azure cluster as well.

Reply:
Changed scope to provided . Tested with qds presto.